docs/download_files.md

```diff
@@ -95,32 +95,51 @@ Every download also includes the individual [QC report](#qc-report) and, if appl
+For [`SingleCellExperiment (R)` downloads](#singlecellexperiment-portal-wide-download-structure), the quantified CITE-seq expression is included in the same file as the gene expression data. 
+For [`AnnData (Python)` downloads](#anndata-portal-wide-download-structure), the quantified CITE-seq expression data is included as a separate file with the suffix `_adt.h5ad`.
 
-#### Download folder structure for individual sample downloads:
-![sample download folder](images/anndata-sample-download-folder.png){width="600"}
+<!--TODO: Update images-->
+### `SingleCellExperiment` Portal-wide download structure
+![portal wide download structure - `sce`](STUB-IMAGE){width="600"}
 
-#### Download folder structure for individual sample downloads with CITE-seq (ADT) data:
-![sample download folder](images/anndata-sample-citeseq-download-folder.png){width="600"}
-
-If downloading a sample that contains a CITE-seq library as an `AnnData` object (`.h5ad` file), the quantified CITE-seq expression data is included as a separate file with the suffix `_adt.h5ad`.
+### `AnnData` Portal-wide download structure
+![portal wide download structure - `anndata`](STUB-IMAGE){width="600"}
 
 <!--TODO:
-spatial section to be moved here, followed by the merged objects section
+spatial section to be moved here
 -->
 
+### Portal wide downloads as merged objects
+
+You can choose to download data from all samples as [merged objects with data from all samples in a single project](#merged-object-downloads) by checking "Merge samples into one object per project". 
+This download includes a folder for each project that contains a single merged object (`SCPCP000000_merged.rds` or `SCPCP000000_merged.h5ad`), a merged summary report (`SCPCP000000_merged-summary-report.html`), a single [metadata](#metadata) file (`single-cell_metadata.tsv`), and all individual [QC reports](#qc-report) and, if applicable, [cell type annotation reports](#cell-type-report) for each library included in the merged object for that project. 
+
+Note that downloading all data using this option _will not_ download a merged object with all samples from all projects, but a single merged object for each project.
+
+<!--TODO: Update images-->
+#### Portal-wide download structure for merged `SingleCellExperiment` objects
+![portal wide download structure - merged `sce`](STUB-IMAGE){width="600"}
+
+#### Portal-wide download structure for merged `AnnData` objects
+![portal wide download structure - merged `anndata`](STUB-IMAGE){width="600"}
+
+### Metadata-only downloads
+
+The Portal-wide metadata download is a single TSV file containing the metadata for all samples with associated single-cell RNA-seq, single-nuclei RNA-seq, or spatial transcriptomics data available on the Portal.
+A table describing all columns included in the file can be found in the [metadata section below](#metadata).
```

Comment:
Do we want to _explicitly_ say bulk-only not included? I'm a bit torn tbh

Reply:
I don't think so because that saves us a doc update later when they are included and it's only 4 samples.